specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.2/keys.json

```diff
@@ -316,6 +316,54 @@
         }
       }
     },
+    "/keys/{key-name}/{key-version}/attestation": {
+      "get": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "GetKeyAttestation",
+        "summary": "Gets the public part of a stored key along with its attestation blob.",
+        "description": "The get key attestation operation returns the key along with its attestation blob. This operation requires the keys/get permission.",
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key to retrieve attestation for."
+          },
+          {
+            "name": "key-version",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Adding the version parameter retrieves attestation blob for specific version of a key. This URI fragment is optional. If not specified, the latest version of the key attestation blob is returned."
```

Comment:
Description says it's optional but `required` is `true`. Which is it? Most other endpoints treat the version as optional, and I think it would be for key attestation.

Reply:
This is optional, Heath. Good catch!. I'll mark required as false.